stylelint.config.js

```diff
@@ -0,0 +1,43 @@
+/** TO DO: use `const browserslist = require('browserslist-config-terra');`
+ *  once browserslist-config-terra module is released.
+ */
+const browserslist = [
+  'iOS >= 10',
+  'last 2 and_chr versions',
+  'last 2 android versions',
+  'last 2 chrome versions',
+  'last 2 edge versions',
+  'last 2 firefox versions',
+  'last 2 ie versions',
+  'last 2 safari versions',
+];
+
+module.exports = {
+  extends: 'stylelint-config-sass-guidelines',
+  plugins: [
+    'stylelint-suitcss',
+    'stylelint-no-unsupported-browser-features',
+  ],
+  rules: {
+    'max-nesting-depth': 3,
+    'scss/at-mixin-pattern': '^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$',
+    'suitcss/custom-property-no-outside-root': true,
+    'custom-property-pattern': [
+      'terra-[a-z]+([a-z0-9-]+[a-z0-9]+)?$',
+      {
+        message: 'Custom property names should be written in lowercase with hyphens',
+      },
+    ],
+    'plugin/no-unsupported-browser-features': [
+      true,
+      {
+        browsers: browserslist,
+        ignore: [
+          'rem', // "rem" is only partially supported by IE 10
+          'flexbox', // "flexbox" is only partially supported by IE 10,11
+        ],
+        severity: 'warning',
+      },
```

Comment:
This plugin as the the ability to ignore rules. Given we use 'rem' and 'flexbox' in several components, it seems these should be disabled. Based on running in clinical, 'word-beak' may be another value to consider. See the error logs

out-of-the-box output:
![screen shot 2018-06-08 at 11 07 38 am](https://user-images.githubusercontent.com/14099737/41169379-739dc156-6b0e-11e8-81c6-bba4ece5d356.png)


with ignore rules defined:
![screen shot 2018-06-08 at 11 15 21 am](https://user-images.githubusercontent.com/14099737/41169389-7869b37a-6b0e-11e8-878e-33cd6cd614aa.png)


**Note:** anytime the browsers list changes we should remove these ignores to verify the same warning is produced.

Reply:
Note: this plugin is a new tool and the config defined here is open to adjustments!

Reply:
Lets add `word-break` to the list